RELEASES.md

```diff
@@ -1,3 +1,123 @@
+Version 1.61.0 (2022-05-19)
+==========================
+
+Language
+--------
+
+- [`const fn` signatures can now include generic trait bounds][93827]
+- [`const fn` signatures can now use `impl Trait` in argument and return position][93827]
+- [Function pointers can now be created, cast, and passed around in a `const fn`][93827]
+- [Recursive calls can now set the value of a function's opaque `impl Trait` return type][94081]
+
+Compiler
+--------
+
+- [Linking modifier syntax in `#[link]` attributes and on the command line, as well as the `whole-archive` modifier specifically, are now supported][93901]
```

Comment:
This also needs a compatibility note.

Previously native static libraries were linked as whole-archive in some cases, now rustc tries to not use whole-archive unless explicitly requested.
This change may result in linking errors in some cases.
To fix such errors native libraries linked from build scripts or `#[link]` attributes need to
- (more common) either be reordered to respect dependencies between them (if `a` depends on `b` then `a` should go first and `b` second)
- (less common) or be updated to use the `+whole-archive` modifier.

Reply:
Relevant docs:
- https://doc.rust-lang.org/nightly/rustc/command-line-arguments.html#-l-link-the-generated-crate-to-a-native-library
- https://doc.rust-lang.org/nightly/rustc/command-line-arguments.html#linking-modifiers-whole-archive
- https://doc.rust-lang.org/nightly/reference/items/external-blocks.html#the-link-attribute

Reply:
Thanks for the thorough writeup!